### DISCUSSION TITLE
how does graphQL group support timezone.

### DISCUSSION THREAD

**Discussion by imiskolee** [directus/directus#15487] [Archive]
hi,team:

As well as we know, we're support graphql and group by datetime field. but date related to local timezone, if our backend always save UTC time, how to support group by date using local timezone?

**Comment by rijkvanzanten** [comment]
I don't understand the question exactly, but if you're storing dates in UTC, you'd want to get the UTC date on the client first, and filter based on that

**Reply by imiskolee** [reply]
> I don't understand the question exactly, but if you're storing dates in UTC, you'd want to get the UTC date on the client first, and filter based on that


save as UTC, but want to group by other timezone(like Hong Kong /Asia)

**Reply by rijkvanzanten** [reply]
There's no way to dynamically alter the timezone of a saved value during grouping/filtering
